adminer/include/design.inc.php

```diff
@@ -121,6 +121,12 @@ function page_footer($missing = "") {
 </div>
 
 <?php switch_lang(); ?>
+<form action="" method="post">
+<div id="hide_menu_button">
+<input type="submit" name="hide_menu" value="&#9776;" id="hide_menu">
```

Comment:
I have set the value at "=" to show the button on my theme. I like this feature !